### ISSUE TITLE
creating new project throws error on /logs/application

### ISSUE THREAD

**Issue by triplepoint** [playframework/play-scala-seed.g8#120]
on a macos machine with:
```
 $ sbt sbtVersion
[warn] No sbt.version set in project/build.properties, base directory: /Volumes/workspace
[info] Set current project to workspace (in build file:/Volumes/workspace/)
[info] 1.2.8
```

doing:
```
sbt new playframework/play-scala-seed.g8
```
with the default values for the two prompts results in:
```
$ sbt new playframework/play-scala-seed.g8
[info] Set current project to workspace (in build file:/Volumes/workspace/)
[info] Set current project to workspace (in build file:/Volumes/workspace/)

This template generates a Play Scala project

name [play-scala-seed]:
organization [com.example]:

Exiting due to error in the template: /var/folders/gj/qnlkvt6x1w579l7jmtp7_pcc0000gn/T/giter8-11860317157110
File: /var/folders/gj/qnlkvt6x1w579l7jmtp7_pcc0000gn/T/giter8-11860317157110/src/main/g8/conf/logback.xml, 7:32: '/logs/application' came as a complete surprise to me
```

**Comment by kynphlee** [comment]
I'm getting the same issue, but in Ubuntu 18.04:

kynphlee@arsen-pc:~/tools/projects/practice/Scala$ sbt new playframework/play-scala-seed.g8
[info] Set current project to scala (in build file:/home/kynphlee/tools/projects/practice/Scala/)
[info] Set current project to scala (in build file:/home/kynphlee/tools/projects/practice/Scala/)

This template generates a Play Scala project 

name [play-scala-seed]: 
organization [com.example]: 

**Exiting due to error in the template: /tmp/giter8-6339976212681
File: /tmp/giter8-6339976212681/src/main/g8/conf/logback.xml, 7:32: '/logs/application' came as a complete surprise to me**

**Comment by henricook** [comment]
I also had this issue, and have raised a PR to correct it: https://github.com/playframework/play-scala-seed.g8/pull/121

Until that is merged you're welcome to use my fork `sbt new henricook/play-scala-seed.g8` - that's an exact clone of the 2.7.x branch here, with the fix applied

**Comment by mkurz** [comment]
Fixed in #121

**Comment by triplepoint** [comment]
Looks good, ran with no errors for me.  Thanks for the quick turnaround.
